File: src/main/java/com/giant/module/auth/repository/custom/AccountRepositoryCustom.java
package com.giant.module.auth.repository.custom;

import com.giant.module.auth.vo.SignInVo;

import java.util.Optional;

public interface AccountRepositoryCustom {
    Optional<SignInVo> findSignInInfoByUserName(String userName);
    Optional<SignInVo> findRefreshInfoByAccountId(Long accountId);
}

File: src/main/kotlin/com/giant/module/auth/repository/custom/AccountRepositoryCustom.kt
package com.giant.module.auth.repository.custom

import com.giant.module.auth.dto.vo.SignInVo

interface AccountRepositoryCustom {
    fun findSignInInfoByUserName(userName:String): SignInVo?
    fun findRefreshInfoByAccountId(accountId:Long): SignInVo?
}